www/content/headers/hx-trigger.md

```diff
@@ -60,9 +60,15 @@ document.body.addEventListener("showMessage", function(evt){
 
 Each property of the JSON object on the right hand side will be copied onto the details object for the event.
 
-Finally, if you wish to invoke multiple events, you can simply add additional properties to the top level JSON
+### Multiple Triggers
+
+If you wish to invoke multiple events, you can simply add additional properties to the top level JSON
 object:
 
 `HX-Trigger: {"event1":"A message", "event2":"Another message"}`
 
+You may as well trigger multiple events with no additional details by sending event names separated by commas, like so:
```

Comment:
Can you change to "You may also"?

Change looks great!

Reply:
Thanks for the correction! I learned [something new](https://dictionary.cambridge.org/grammar/british-grammar/also-as-well-or-too)

Reply:
Just in case you had missed it, allow me to ping you @1cg as I know you're overwhelmed with notifications/email